agent/src/agent/data_extractor/topology/lookup.py

```diff
@@ -0,0 +1,58 @@
+import time
+
+from agent.data_extractor.topology import entity
+from agent.modules import constants
+
+
+def _get_expiration_time():
+    return time.time() + constants.LOOKUP_CACHE_TTL_SECONDS
+
+
+class Cache:
```

Comment:
just wanna say, nice implementation with the magic methods :)

Reply:
thanks :)